server/sonar-ce/src/main/java/org/sonar/ce/platform/ComputeEngineExtensionInstaller.java

```diff
@@ -19,14 +19,27 @@
  */
 package org.sonar.ce.platform;
 
+import com.google.common.collect.ImmutableSet;
+import java.lang.annotation.Annotation;
+import java.util.Collection;
+import java.util.Collections;
 import org.sonar.api.SonarRuntime;
 import org.sonar.api.ce.ComputeEngineSide;
+import org.sonar.api.config.Settings;
 import org.sonar.api.server.ServerSide;
 import org.sonar.core.platform.PluginRepository;
 import org.sonar.server.plugins.ServerExtensionInstaller;
 
 public class ComputeEngineExtensionInstaller extends ServerExtensionInstaller {
-  public ComputeEngineExtensionInstaller(SonarRuntime sonarRuntime, PluginRepository pluginRepository) {
-    super(sonarRuntime, pluginRepository, ServerSide.class, ComputeEngineSide.class);
+  public ComputeEngineExtensionInstaller(SonarRuntime sonarRuntime, PluginRepository pluginRepository, Settings settings) {
+    super(sonarRuntime, pluginRepository, getSupportedAnnotations(settings));
   }
+
+   private static Collection<Class<? extends Annotation>> getSupportedAnnotations(Settings settings) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)